src/components/Main/Features/Features.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import data from '../../../../data'
+import PropTypes from 'prop-types';
+
+export const Features = (props) => {
+
+  const {
+    className,
+    features,
+    ...otherProps
+  } = props
+
+  const renderListItem = (item, i) => {
+    return (
+      <>
```

Comment:
Skoro mamy jedno dziecko to może ono być tym elementem nadrzędnym i nie ma potrzeby robić "fragmentu (<>)"

PS. Brakuje `key` w rodzicu